### ISSUE TITLE
Error running `tiffutil`! Exit code was 5

### ISSUE THREAD

**Issue by stormtea123** [loopline-systems/electron-builder#45]
```
npm run pack:osx
```
`package.json`
```
{
  "name": "hummingbird",
  "version": "1.0.0",
  "description": "",
  "main": "main.js",
  "scripts": {
    "dev": "electron ./app",

    "clean": "rm -rf ./dist",
    "clean:osx": "rm -rf ./dist/osx",
    "clean:win": "rm -rf ./dist/win",

    "build": "npm run clean && npm run build:osx && npm run build:win",
    "build:osx": "npm run clean:osx && electron-packager ./app \"Loopline Systems\" --out=dist/osx --platform=darwin --arch=x64 --version=0.25.3 --icon=assets/osx/icon.icns",
    "build:win": "npm run clean:win && electron-packager ./app \"Loopline Systems\" --out=dist/win --platform=win32 --arch=ia32 --version=0.25.3 --icon=assets/win/icon.ico",

    "pack": "npm run pack:osx && npm run pack:win",
    "pack:osx": "npm run build:osx && electron-builder \"dist/osx/Loopline Systems.app\" --platform=osx --out=\"dist/osx\" --config=packager.json",
    "pack:win": "npm run build:win && electron-builder \"dist/win/Loopline Systems-win32\" --platform=win --out=\"dist/win\" --config=packager.json"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/stormtea123/hummingbird.git"
  },
  "author": "stormtea123",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/stormtea123/hummingbird/issues"
  },
  "homepage": "https://github.com/stormtea123/hummingbird#readme",
  "dependencies": {
    "imagemin": "^3.2.0",
    "imagemin-mozjpeg": "^5.1.0",
    "imagemin-pngquant": "^4.2.0",
    "imagemin-webp": "^3.1.0",
    "nconf": "^0.8.2"
  },
  "devDependencies": {
    "electron-packager": "^4.0.2",
    "electron-prebuilt": "^0.25.2",
    "electron-builder": "^1.0.0"
  }
}
```
`packager.json`
```
{
  "osx" : {
    "title": "Hummingbird",
    "background": "/Library/WebServer/Documents/hummingbird/assets/osx/background.png",
    "icon": "assets/osx/icon.icns",
    "icon-size": 80,
    "contents": [
      { "x": 200, "y": 20, "type": "link", "path": "/Applications" },
      { "x": 20, "y": 20, "type": "file" }
    ]
  },
  "win" : {
    "title" : "Hummingbird",
    "icon" : "assets/win/icon.ico"
  }
}
```

```
- Starting build for ´osx´ -
Writing temporary ´appdmg.json´
Wrote temporary ´appdmg.json´
Kicking off ´appdmg´
appdmg: [1] Looking for target
appdmg: [2] Reading JSON Specification
appdmg: [3] Parsing JSON Specification
appdmg: [4] Validating JSON Specification
appdmg: [5] Looking for files
appdmg: [6] Calculating size of image
appdmg: [7] Creating temporary image
appdmg: [8] Mounting temporary image
appdmg: [9] Making hidden background folder
appdmg: [10] Copying background
appdmg: [11] Unmounting temporary image
appdmg: [12] Removing temporary image
/Library/WebServer/Documents/hummingbird/node_modules/electron-builder/cli.js:30
    throw error;
          ^
Error: ENOENT, open '/Library/WebServer/Documents/hummingbird/Library/WebServer/Documents/hummingbird/assets/osx/background.png'
    at Error (native)

npm ERR! Darwin 14.5.0
npm ERR! argv "/Users/admin/.nvm/versions/node/v0.12.0/bin/node" "/Users/admin/.nvm/versions/node/v0.12.0/bin/npm" "run" "pack:osx"
npm ERR! node v0.12.0
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE
npm ERR! hummingbird@1.0.0 pack:osx: `electron-builder "dist/osx/Hummingbird.app" --platform=osx --out="dist/osx" --config=packager.json`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the hummingbird@1.0.0 pack:osx script 'electron-builder "dist/osx/Hummingbird.app" --platform=osx --out="dist/osx" --config=packager.json'.
npm ERR! This is most likely a problem with the hummingbird package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     electron-builder "dist/osx/Hummingbird.app" --platform=osx --out="dist/osx" --config=packager.json
npm ERR! You can get their info via:
npm ERR!     npm owner ls hummingbird
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Library/WebServer/Documents/hummingbird/npm-debug.log
```

**Comment by stefanjudis** [comment]
Please follow contribution guideline. :)

https://github.com/loopline-systems/electron-builder/blob/master/CONTRIBUTING.md

**Comment by stormtea123** [comment]
@stefanjudis  I have update

**Comment by stefanjudis** [comment]
@stormtea123 

Hey hey, thanks for the update.
First thing I notice is, that you're including the `electron-builder` inside of your distribution build.
This shouldn't be the case, or? 

Can you please ignore modules that are not needed via `electron-packager`?
After that - we can check the what's going on with `appdmg` there, deal? :bowtie: 

Docs to packager: https://github.com/maxogden/electron-packager#usage

**Comment by stormtea123** [comment]
@stefanjudis Thank you.
`electron-packager` no problem,I have resolved with https://github.com/LinusU/node-appdmg ,
[hummingbird](https://github.com/stormtea123/hummingbird/releases) is builder I use `node-appdmg`.

**Comment by stefanjudis** [comment]
Sorry I don't understand that?

all good now?

**Comment by stormtea123** [comment]
I resolved with [node-appdmg](https://github.com/LinusU/node-appdmg) , Not with `electron-builder`

**Comment by stefanjudis** [comment]
So we can close this here? 
You resolved the issue?

If you have no problems anymore - please close the issue.

**Comment by stormtea123** [comment]
'electron-builder' will still appear above error, I expect you to fix it. Temporarily use `node-appdmg` substitute 'electron-builder',do you understand me? :bowtie:

**Comment by stefanjudis** [comment]
Can you please answer my questions then first?

> Can you please ignore modules that are not needed via electron-packager?
> After that - we can check the what's going on with appdmg there, deal? :bowtie:

**Comment by stormtea123** [comment]
@stefanjudis  Sorry, my english is not very good,  i'm so sorry i didn't make myself clear.
 I have update `package.json`
`ignore modules that are not needed via electron-packager`

**Comment by stefanjudis** [comment]
That's no problem at all. 
Can you please give me the link to your project again?

( I assume you deleted your comments? )

**Comment by stormtea123** [comment]
I've updated my [project](https://github.com/stormtea123/hummingbird)

**Comment by stefanjudis** [comment]
@stormtea123 

I'm pretty sure you have a different state on github than on your local machine?
The `packager.json` on github does not include `background.png`.

**Comment by stefanjudis** [comment]
@stormtea123 I think it's fixed with version `v2.0.2`. Can you please confirm?

If not feel free to reopen.

**Comment by stormtea123** [comment]
Thank you, `v2.0.2` is ok.
